dropboxfs.py

```diff
@@ -133,6 +133,7 @@ def __init__(self, client, name):
         self.closed = False
         self.pos = 0
         self.seek_pos = 0
+        super(ChunkedReader, self).__init__(self.r, name)
```

Comment:
Not a big deal but it probably makes more sense to move this up so that it's the first instruction of `__init__` in order to avoid that the attrs set earlier get inadvertently overwritten.